modin/data_management/factories/dispatcher.py

```diff
@@ -84,99 +127,123 @@ def _update_engine(cls, _):
             cls.__engine.prepare()
 
     @classmethod
-    def from_pandas(cls, df):
+    def from_pandas(cls, df):  # noqa: PR01
```

Comment:
`PR01: parameters are not documented`. Roles of the parameters described in the docstring itself.

Reply:
I would rather we use the format template where possible instead of silencing the error checks.